eml/components/general/IconHeader.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { View, Image } from 'react-native';
+import Text from './Text';
+import PropTypes from 'prop-types';
+
+/**
+ * Custom header component with an icon and title.
+ * @param {string} title - The title to display next to the icon.
+ * @returns {JSX.Element} The IconHeader component.
+ */
+export default function IconHeader({ title, description }) {
+
+  IconHeader.propTypes = {
+    title: PropTypes.string,
+    description: PropTypes.string,
+  };
+
+    return (
+        <View>
+        <View className="flex flex-row items-center pl-6 pb-2 pt-[20%]">
+            
+            <Image
+                source={require('../../assets/singleIcon.png')}
```

Comment:
The path here is not correct, it should be "require('../../assets/images/singleIcon.png')"